packages/notebook-extension/schema/tracker.json

```diff
@@ -774,6 +774,12 @@
       "description": "Defines the maximum number of output cells to to to be rendered in the output area for cells with many outputs. The output area will have a head and a tail, and the outputs between will be trimmed and not displayed unless the user clicks on the information message.",
       "type": "number",
       "default": 50
+    },
+    "enableDocumentWideUndoRedo": {
+      "title": "Enable the undo/redo (`z` keystroke) on the notebook document level.",
+      "description": "Enable the undo/redo (`z` keystroke) on the notebook document level, so actions like cells move, change... can have history. The undo/redo never applies on the outputs, in other words, outputs don't have history.",
```

Comment:
A general/comment-question: is the `z` keystroke singled out for a specific reason? I think that the document-wide undo/redo pertains both <kbd>z</kbd> keystroke and the <kbd>Ctrl</kbd> + <kbd>z</kbd> shortcut (/Mac equivalent).

Reply:
If this works for you , and to avoid confusion with upcoming PR, I will remove mention of z/ctrl+z and will just keep the name "document-wide history"